websites/D/Drama/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Drama",
+    "author": {
+      "name": "Toki",
+      "id": "401067750354845727"
+    },
+    "category": "socials & stream",

```

Comment:
This category does not exist in PreMiD. 
Categories: https://docs.premid.app/en/dev/presence/metadata#presence-categories